File: Java/app/src/main/java/com/omeryildizce/java/Loops.java
package com.omeryildizce.java;

public class Loops {
    public static void main(String[] args) {

        int[] myNumbers = {2, 3, 5, 7, 11, 13, 17, 19, 23, 27, 29};
        for (int i = 0; i < myNumbers.length; i++) {
            System.out.println(myNumbers[i]);
        }

        for (int number : myNumbers) {
            System.out.println();
        }


        int j = 0;
        while (j < 10000000) {
            System.out.println(j);
            j++;
        }
    }
}

File: s15_KotlinLearning/KotlinLearning/app/src/main/java/com/omeryildizce/kotlinlearning/kotlin/Loops.kt
package com.omeryildizce.kotlinlearning.kotlin


fun main(args: Array<String>) {
    val array = Array(30) { i -> (i * 3) + 3 }

    for (number in array) {
        println(number)
    }

    val array1 = arrayOf(1..25 step 2)
    println(array1[0])

    var a = 0
    while (a<10){
        a++
        println(a)
    }
}